lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -261,6 +261,26 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
           currentValue: adoptOpenJreMatches.version,
         };
       }
+      const semeruJdkMatches = version.match(
+        /^semeru-openj9-(?<version>\d\S+)_openj9-(?<openj9>\d\S+)/
+      )?.groups;
+      if (semeruJdkMatches) {
+        return {
+          datasource: JavaVersionDatasource.id,
+          packageName: 'java-jdk',
+          currentValue: semeruJdkMatches.version,
+        };
+      }
+      const semeruJreMatches = version.match(
+        /^semeru-jre-openj9-(?<version>\d\S+)_openj9-(?<openj9>\d\S+)/
```

Comment:
```suggestion
        /^semeru-jre-openj9-(?<version>\d\S+)_openj9-\d\S+/
```
don't capture openj9, makes no sense because not used

Reply:
On this, how will Renovate handle different versions of OpenJ9? 

Will it always pick the latest? Or an unexpected one?

Should we do anything to cater for it, what in that case?